codegens/r-httr/test/newman/newman.test.js

```diff
@@ -13,7 +13,7 @@ describe('r-httr Converter', function () {
         fileName: 'snippet.r',
         // Run script required to run the generated code snippet
         runScript: 'Rscript snippet.r',
-        skipCollections: ['redirectCollection', 'unsupportedMethods']
+        skipCollections: ['redirectCollection', 'unsupportedMethods', 'rawBody']
```

Comment:
@webholik I didn't get why we're skipping this collection now after adding it. Could you explain?

Reply:
@VShingala R-HTTR also has the same problem as the `curl` codegen of unnecessarily escaping the `\` character. But since its usage is low and no user has reported it till now, we can tackle it in separate PR as part of regular bug fixes.